asskey-pass.yml

```
@@ -0,0 +1,125 @@
+          env:
+            -GH_TOKEN: ${ secrets.GITHUB_TOKEN }
+        - name: Setup SBOM Tool
+          run: |
+            curl -lo #https://github.com/microsoft/sbom-tool?tab=readme-ov-file#sbom-generation
+            codeql database build --language=java --source-root=attestation-exploit/security-pubring/**.ql    
+        - name: Generate SBOM
+          run: |
+            ./sbom-tool generate -o sbom-output.xml 
+            sbom-tool validate sbom-output.xml  
+          if: always() == sbom-tool redact --input sbom-output.xml --output ./METADATA-FTP
+          run: |   
```

YAMLlint (1.38.0)

[error] duplication of key "run" in mapping

(key-duplicates)